api/build.gradle.kts

```diff
@@ -16,6 +16,16 @@ android {
         sourceCompatibility = JavaVersion.VERSION_1_8
         targetCompatibility = JavaVersion.VERSION_1_8
     }
+
+    buildTypes {
+        debug {
+            buildConfigField("String", "API_ID", "\"63c5573a496ae06190e061d3\"")
```

Comment:
ça serait mieux de ne pas divulguer ton api_key. Tu peux checker cet article : https://guides.codepath.com/android/Storing-Secret-Keys-in-Android

Reply:
Super!